.github/workflows/release-drafter-draft-release.yml

```diff
@@ -0,0 +1,36 @@
+name: Create Draft Release (GitHub only)
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'tag/version'
+        required: true
+        default: '1.0.0'
+
+      action_tag:
+        description: 'create tag ("no" to skip)'
+        required: true
+        default: 'yes'
+
+      create_as_draft:
+        description: 'create as draft release? ("no" to create real release)'
+        required: true
+        default: 'yes'
+
+jobs:
+  create_release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: release-drafter/release-drafter@v5
+      if: ${{ github.event.inputs.action_tag == 'yes' }}
+      with:
+        name: v${{ github.event.inputs.tag }}
+        tag: ${{ github.event.inputs.tag }}
+        version: ${{ github.event.inputs.tag }}
+        publish: ${{ github.event.inputs.create_as_draft == 'no' }}
+        config-name: release-drafter-draft-release-config.yml
```

Comment:
if there are no multiple files required for this action, can we use the default: `release-drafter.yml`?